### PR TITLE
Added firebase for scaling

### PR DESCRIPTION
**Pull request by philipdalen**

### CODE REVIEW

File: server/test/load-test.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+import url from 'url';
+import axios from 'axios';
+const socketio = require('@feathersjs/socketio-client');
+const io = require('socket.io-client');
+import feathers from '@feathersjs/feathers';
+import { getFeahtersToken }  from './setup-tests/test-set'
+import app from '../src/app';
+
+const port = app.get('port') || 8998;
+const getUrl = (pathname?: string): string => url.format({
```
Comment (StevenMalaihollo) [inline server/test/load-test.ts:13]: unused?

File: server/test/firestore-tests-.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
+import 'mocha';
+import { assert, use } from 'chai';
+import { Server } from 'http';
+import url from 'url';
+import feathers from '@feathersjs/feathers';
+import { getFeahtersToken }  from './setup-tests/test-set'
```
Comment (StevenMalaihollo) [inline server/test/firestore-tests-.ts:6]: Could you remove all this unused imports, comments

File: server/test/load-test.ts [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+// to be present
+// -----------------------------------------------------------------------------------------
+
+describe('Feathers application server tests', () => {
+    let server: Server;
+    const testingVariables = app.get('testingSet')
+    const host = app.get('host')
+    const protocol = app.get('protocol')
+
+  it.only('Perform a socket load test on an environment', async (done) => {
```
Comment (StevenMalaihollo) [inline server/test/load-test.ts:34]: Should be skipped

File: server/test/load-test.ts [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+// -----------------------------------------------------------------------------------------
+// NB!!! Before running this performance test you have to run
+// npm run mocha-dev-in-cloud
+// this will set the NODE_ENV to dev-in-cloud which will use the dev-in-cloud config file
+// in the config file you will notice the performacnce test are reliying on hard coded data
+// to be present
+// -----------------------------------------------------------------------------------------
+
+describe('Feathers application server tests', () => {
```
Comment (StevenMalaihollo) [inline server/test/load-test.ts:28]: Change name?

**Comment by StevenMalaihollo** [comment]
Conflicts

**Review by StevenMalaihollo — APPROVED** [review]

File: server/test/load-test.ts (hunk 1 above)
Comment (philipdalen) [reply server/test/load-test.ts:13]: resolved

File: server/test/load-test.ts (hunk 4 above)
Comment (philipdalen) [reply server/test/load-test.ts:28]: done

File: server/test/load-test.ts (hunk 3 above)
Comment (philipdalen) [reply server/test/load-test.ts:34]: done

File: server/test/firestore-tests-.ts (hunk 2 above)
Comment (philipdalen) [reply server/test/firestore-tests-.ts:6]: done

**Pull request merged**